frontend/src/lib/UriUtil.ts

```diff
@@ -33,36 +37,56 @@ export function getWindowBaseUriParts(): BaseUriParts {
   // server's port 3000.
   // If changed, also change config.py
   const host = window.location.hostname
+
+  // prettier-ignore
   const port = IS_DEV_ENV
     ? WEBSOCKET_PORT_DEV
     : window.location.port
-    ? Number(window.location.port)
-    : isHttps()
-    ? 443
-    : 80
-  return { host, port }
+      ? Number(window.location.port)
+      : isHttps()
+        ? 443
+        : 80
```

Comment:
Unrelated to the PR, but I simply had to fix this. This stuff was unreadable before :(